owncloud/core
# Support for CORS on OCS and Webdav APIs + domain whitelisting

## Description

Manually added the * header

## Motivation and Context

Trying to enable CORS.

## Testing instructions:
1. Download this file: https://github.com/noveens/js-owncloud-client/blob/master/browser/bundle.js
2. Create a html file in the same directory with the following code.
```html
<!DOCTYPE html>
<html>
	<head>
		<title>owncloud Lib. test</title>
		<script type="text/javascript" src="./bundle.js"></script>

		<script type="text/javascript">
			// var oc is global
			oc.setInstance('PUT OC INSTANCE HERE');
			
			oc.login('USERNAME', 'PASSWORD').then(status => {
				alert(status);
			}).catch(error => {
				alert(error);
			});

		</script>
	</head>

	<body>
		Check the console.
	</body>
</html>
```

3. Try sniffing the request to see it's response headers (Can be done using the Network tab on the right to Console in browser)

@PVince81
